WordPress/Classes/Utility/Analytics/WPAnalyticsTrackerAutomatticTracks.m

```diff
@@ -1991,6 +1991,18 @@ + (TracksEventPair *)eventPairForStat:(WPAnalyticsStat)stat
             eventName = @"widget_active_site_changed";
             break;
 
+        case WPAnalyticsStatWelcomeNoSitesInterstitialShown:
```

Comment:
Like mentioned in chat, Everything related to the events should have been in a separate PR